.github/workflows/netlify.yml

```diff
@@ -53,7 +43,7 @@ jobs:
           deploy-message: "Deploy from GitHub Actions"
           enable-pull-request-comment: false
           enable-commit-comment: false
-          alias: pr${{ steps.readctx.outputs.prnumber }}
+          alias: pr${{ steps.prdetails.outputs.pr_id }}
```

Comment:
this is maybe more of a comment on the action, but this doesn't seem to be defined in the action's documentation. I'd really rather the action be properly documented before we switch things over to it.

Reply:
Good call, I tend to rely on documenting within actions.yml but duplicating to README helps - https://github.com/matrix-org/pr-details-action/blob/main/README.md#inputs